pkgs/by-name/ts/ts-warp/package.nix

```
@@ -0,0 +1,34 @@
+{
+  fetchFromGitHub,
+  lib,
+  stdenv,
+  which,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "ts-warp";
+  version = "1.5.4";
+
+  src = fetchFromGitHub {
+    owner = "mezantrop";
+    repo = "ts-warp";
+    rev = finalAttrs.version;
+    hash = "sha256-iozjUHdiiP9Wg86W/LGUucd3sLC8gHMcYAbyAHT7WrQ=";
+  };
+
+  nativeBuildInputs = [ which ];
+
+  PREFIX = "$(out)";
+
+  installFlags = [ "PREFIX=$(out)" ];
```

Cant this be passed only once as `makeFlagsArray`?
```suggestion
  makeFlagsArray = [
    "PREFIX = ${placeholder "out"}"
  ];
```

---

For some reason, the configure phase always tries checking for the default PREFIX (which is `/usr/local /opt $HOME`) which doesn't exist on NixOS and totally ignores the `makeFlagsArray` `$PREFIX`. So the check phase fails on NixOS if I don't set the `$PREFIX` environment variable the way I did. Is there a better way to set an environment variable?

---

I believe `env.PREFIX = "$(out)";` is the recommended way then